app/Services/ReportbackService.php

```diff
@@ -34,6 +35,38 @@ public function create($data)
     {
         $reportback = $this->reportbackRepository->create($data);
 
+        // POST reportback back to phoenix.
+        $body = [
+            'uid' => $reportback->drupal_id,
+            'nid' => $reportback->campaign_id,
+            'quantity' => $reportback->quantity,
+            'why_participated' => $reportback->why_participated,
+            'file_url' => $reportback['items'][0]['file_url'],
+            'caption' => $reportback['items'][0]['caption'],
+            'source' => $reportback['items'][0]['source'],
+        ];
+
+        $phoenixResponse = $this->phoenix->postReportback($reportback->campaign_id, $body);
+
+        // If POST to Phoenix fails, record in failed_logs table.
```

Comment:
awesome, very cool, @DFurnes ! @angaither should I do this for this PR or in interest of getting this to keep moving, make an issue for this to iterate on this next?